doc/changelog.md

```diff
@@ -203,32 +212,48 @@
   4.0
   ```
 
-  ```py
-  from catalyst.debug import get_pipeline_output, replace_ir
+  Here we use `get_compilation_stage` to acquire the IR, and then modify
+  `%2 = arith.mulf %in, %in_0 : f64` to turn the square function into a cubic one
+  via `replace_ir`:
+
+  ```python
+  from catalyst.debug import get_compilation_stage, replace_ir
 
-  old_ir = get_pipeline_output(f, "HLOLoweringPass")
+  old_ir = get_compilation_stage(f, "HLOLoweringPass")
   new_ir = old_ir.replace(
       "%2 = arith.mulf %in, %in_0 : f64\n",
       "%t = arith.mulf %in, %in_0 : f64\n    %2 = arith.mulf %t, %in_0 : f64\n"
   )
   replace_ir(f, "HLOLoweringPass", new_ir)
   ```
 
+  The recompilation starts after the given checkpoint stage:
+
   ```pycon
   >>> f(2.0)
-   8.0
+  8.0
   ```
 
 <h3>Improvements</h3>
 
-* Eliminate (some) scalar tensors from the IR by adding a `linalg-detensorize` pass at the end of the HLO lowering passes.
-  [(#1010)](https://github.com/PennyLaneAI/catalyst/pull/1010)
+* Catalyst has been updated to support JAX v0.4.28.
+  [(#931)](https://github.com/PennyLaneAI/catalyst/pull/931)
+  [(#995)](https://github.com/PennyLaneAI/catalyst/pull/995)
 
 * Catalyst now supports keyword arguments for qjit-compiled functions.
   [(#1004)](https://github.com/PennyLaneAI/catalyst/pull/1004)
 
-* Catalyst is now compatible with Enzyme `v0.0.130`
-  [(#898)](https://github.com/PennyLaneAI/catalyst/pull/898)
+  ```pycon
+  >>> @qjit
+  ... @grad
+  ... def f(x, y):
+  ...     return x * y
+  >>> f(3., y=2.)
+  Array(2., dtype=float64)
+  ```
+
+  Note that the `static_argnums` argument to the `qjit` decorator
+  is not supported when passing argument values as keyword arguments.
```

Comment:
Something we should add to the backlog!

Reply:
Is this supported now that we fixed keyword argument support in #1004 ?

Reply:
yep!